<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1742</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>0</y>
      <width>1011</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>40</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Map</string>
    </property>
    <widget class="QPushButton" name="pushButtonClearMap">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>140</y>
      <width>671</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Communication</string>
    </property>
    <widget class="QTextEdit" name="textEditLogs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>651</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditText2Send">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>561</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSend">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>380</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClear">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>30</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1742</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
